src/PIL/Image.py

```diff
@@ -767,7 +767,7 @@ def tobytes(self, encoder_name="raw", *args):
 
         data = []
         while True:
-            l, s, d = e.encode(bufsize)
+            l, s, d = e.encode(bufsize)  # noqa: E741
```

Comment:
Please can you check what `l` represents? Is there a better name?

Reply:
`l`, `s`, and `d` are the length of the encoded data, an error code, and the encoded data.